source/Junkership.js

```diff
@@ -4,69 +4,74 @@ var Keyb = require("keyb")
 import Reference from "./Reference.js"
 import Projectile from "./Projectile.js"
 import Score from "./Score.js"
-import {PeaShoota, PowerUp, TriShoota, FiveShoota, RapidFire, SprayShot,SuperSprayShot, CrazySprayShot, VertSprayShot, VertShoota, RapidSprayShot, BFG} from "./PowerUp.js"
-
+import {PeaShoota, PowerUp, TriShoota, FiveShoota, RapidFire, SprayShot,
```

Comment:
OOC: Does the `import * from "./PowerUp.js"` not work with our current build system?